_posts/2017-10-05-apprenticeship-at-8thlight.md

```diff
@@ -0,0 +1,213 @@
+
+[impostor syndrome]: https://en.wikipedia.org/wiki/Impostor_syndrome
+
+## The Apprenticeship
+
+The resident apprentice focuses on “programming, client services, collaboration, leadership, and communication skills” 
+as their [career site] states. Once graduated a crafter has obtained the skills to “learn to lead by example, provide and receive honest feedback, and answer the questions a client doesn't ask”.
+
+[career site]: https://8thlight.com/job-opportunities/london-crafter/
+
+The **apprenticeship is run like a real project**, like any other client project at 8th Light. That is the first learning: discipline. By running the apprenticeship like a project an apprentice lives and breathes all parts of a project iteration from the beginning and won’t be surprised when working with customers later.
+
+At the beginning the mentor creates a curriculum. (At some point I remember the ThoughtWorks learning path was mentioned too.) The curriculum might be a simple spread sheet that mentor and apprentice have access too. Mentor - that is the person that guides the apprentice through their apprenticeship, more about that later. The curriculum is the high-level plan for the apprenticeship and every week (that’s how long iteration take) more detailed tasks that are derived from the curriculum are put into a Kanban-like board. Theses tasks get estimated by the apprentice and depending on the acceptable load the next iteration’s “to-do” column is being filled. The board that we saw being used at 8th Light was a simple three column board with the columns backlog, to-do and done.
+
+At the beginning of an apprenticeship, at the end and also while it is running the apprentices are asked to do a self-assessment on all the topics that the apprenticeship will cover.
+
+## The Curriculum
+
+As described above the curriculum contains all things about “programming, client services, collaboration, leadership, and communication skills”. The curriculum itself is way more detailed than this and contains the topics in detail, (chapters of) books to read, blog posts, tasks to solve and so on.
+
+One thing I saw that everyone at 8th Light has as part of their apprenticeship is to learn the editor vi and the terminal manager tmux. Other tools are allowed to use too, but as one part of the curriculum those two tools are required to learn. Another topic that appears in the curriculum is Test Driven Development. I remember Mollie saying that the “disciplined TDD” was one of the most important things she had learned in her apprenticeship. On [her blog][tests-that-test-you] Katerina writes about about testing on the third day of her apprenticeship. This makes obvious that testing is not added into the game at some later stage to validate something that was implemented. It is an essential part, a building block of an apprenticeship. I heard very often that the book [TDD by Example] was mentioned and is part of an apprenticeship.
```

Comment:
I don't think that it's explicit that everyone use vim + tmux, though I do think everyone knows their way around. We have some emacs users as well, so we're not discriminatory about tooling.

Reply:
:)